migration/v0.43/apply/10-kube-prometheus-stack.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    chart_version=$(yq4 '.version' "${ROOT}/helmfile.d/upstream/prometheus-community/kube-prometheus-stack/Chart.yaml")
+    clusters=("${CK8S_CLUSTER}")
+    if [[ "${CK8S_CLUSTER}" == "both" ]]; then
+      clusters=("wc" "sc")
+    fi
+
+    for cluster in "${clusters[@]}"; do
+      current_version=$(helm_do "${cluster}" get metadata -n monitoring kube-prometheus-stack -ojson | jq '.version' | tr -d '"')
```

Comment:
Suggestion: Use the raw output flag in jq to get without quotes
```suggestion
      current_version=$(helm_do "${cluster}" get metadata -n monitoring kube-prometheus-stack -ojson | jq -r '.version')
```